bin/watch

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
+yml='-f docker-compose.yml -f docker-compose.watch.yml'
+
+trap "echo ' ...Ctrl+C'; docker kill pattern-library-gulp-watch > /dev/null; docker rm -v pattern-library-gulp-watch > /dev/null; docker kill pattern-library-php-watch > /dev/null; docker rm -f pattern-library-php-watch > /dev/null; docker-compose $yml down;" SIGINT 
```

Comment:
This seemed not to work for me.

Reply:
What's the reaction to Ctrl+C, anything being printed? Can try with the Mac test mule later

Reply:
No:
```bash
bash-4.4$ bin/watch
Starting assets-builder Docker container...
pattern-library-gulp-watch
Starting ui Docker container...
Starting patternlibrary_assets-builder_1 ... done
Starting patternlibrary_assets_1         ... done
configuring pattern lab...
your site has been generated...
site generation took 5.1034440994263 seconds and used 20MB of memory...
Starting patternlibrary_assets-builder_1 ... done
Starting patternlibrary_assets_1         ... done
pattern-library-php-watch
Starting patternlibrary_assets-builder_1 ... done
Starting patternlibrary_assets_1         ... done
Starting patternlibrary_ui-builder_1     ... done
Starting patternlibrary_ui_1             ... done
bash-4.4$ [12:59:24] Using gulpfile /srv/pattern-library/gulpfile.js
[12:59:24] Starting 'sass:watch'...
[12:59:26] Finished 'sass:watch' after 1.96 s
[12:59:26] Starting 'img:watch'...
[12:59:38] Finished 'img:watch' after 12 s
[12:59:38] Starting 'js:watch'...
[12:59:39] Finished 'js:watch' after 1.21 s
[12:59:39] Starting 'fonts:watch'...
[12:59:39] Finished 'fonts:watch' after 19 ms
[12:59:39] Starting 'watch'...
[12:59:39] Finished 'watch' after 5.23 μs
configuring pattern lab...
your site has been generated...
site generation took 5.6629750728607 seconds and used 20MB of memory...
watching your site for changes...
^C
bash-4.4$
```
and http://localhost:8889/ is still accessible.